omnibus/resources/agent/msi/source.wxs.erb

```diff
@@ -720,8 +644,34 @@
     <WixVariable Id="WixUIDialogBmp" Value="Resources\assets\dialog_background.bmp" />
     <WixVariable Id="WixUIBannerBmp" Value="Resources\assets\banner_background.bmp" />
 
-    <CustomAction Id='SetFinalizeProperty' Return='check' Property='FinalizeInstall' 
-        Value='PROJECTLOCATION=[PROJECTLOCATION];APPLICATIONDATADIRECTORY=[APPLICATIONDATADIRECTORY];DDAGENTUSER_NAME=[DDAGENTUSER_NAME];DDAGENTUSER_PASSWORD=[DDAGENTUSER_PASSWORD];SYSPROBE_PRESENT=[SYSPROBE_PRESENT];ADDLOCAL=[ADDLOCAL]' />
+    <!-- Properties must each be on their own line and end with a semicolon ; -->
+    <CustomAction Id='SetFinalizeProperty' Return='check' Property='FinalizeInstall'
+        Value='
```

Comment:
I'm aware this was my suggestion, but this should have a comment that
a) this causes a compiler warning
b) the warning is on the base string, not the expanded string (and therefore not that helpful, anyway)
c) there appears to be no length limit at runtime